lib/phlex/sgml.rb

```diff
@@ -223,6 +223,19 @@ def render(renderable = nil, &)
 		nil
 	end
 
+	def cache(*keys, **, &)
```

Comment:
We should require at least one key here. Alternatively, if no key is provided, we could use `caller_locations[0].lineno`.